package.json

```diff
@@ -130,11 +131,8 @@
     "redux-thunk": "^2.3.0",
     "redux-undo": "^1.0.1",
     "reselect": "^4.1.5",
-    "styled-components": "^5.2.0",
-    "stylelint-config-standard-scss": "^3.0.0",
-    "stylelint-processor-styled-components": "^1.10.0",
-    "url": "^0.11.0",
-    "uuid": "^8.3.0"
```

Comment:
is it never used in source code?

Reply:
I didn't find usage of it. I found only such code in unit tests `jest.mock('uuid', () => mockUuid)`

Reply:
looks as we do use it, ex. in src\models\Label.js

Reply:
Oh, I'm sorry, missed it.